content/essentials/intro-to-reactivity.mdx

```diff
@@ -127,9 +128,8 @@ Unlike dealing with individual signals, stores offer a more organized way to man
 
 #### State management
 
-At their core, stores serve as containers for state. 
-They facilitate centralized management of state by offering methods to update, reset, or manipulate it. 
-This not only improves the predictability of state transitions but simplifies the process of interacting with and modifying state.
+At their core, stores serve as containers for state, facilitating centralized state management by providing methods for updates and manipulations.
+This improves state transitions by streamlining the process of interacting with and altering state.
```

Comment:
This sentence seems pretty confusing to me, I'm not sure what its saying. And "to alter" is not a thing that people say about state I think, like I've never heard it use elsewhere that I can remember anyway.